test/e2e_node/security_context_test.go

```diff
@@ -373,6 +373,14 @@ var _ = framework.KubeDescribe("Security Context", func() {
 				if !isSupported {
 					framework.Skipf("Skipping because no_new_privs is not supported in this docker")
 				}
+				// It turns out SELinux policy in RHEL 7 does not play well with
+				// the "NoNewPrivileges" flag. So let's skip this test when running
+				// with SELinux support enabled.
```

Comment:
Could we add a link to the upstream issue and perhaps add some TODO item to remove this code when it will be fixed in the upstream?

Reply:
Done. PTAL